tools/go-agent/config/agent.default.yaml

```diff
@@ -30,8 +30,11 @@ agent:
   correlation:
     max_key_count: ${SW_AGENT_CORRELATION_MAX_KEY_COUNT:3}
     max_value_size: ${SW_AGENT_CORRELATION_MAX_VALUE_SIZE:128}
-  # If the operation name of the first span is included in this set, this segment should be ignored.(multiple split by ",")
+  # If the operation name of the first span is included in this set, this segment should be ignored.(multiple split by ",").
   ignore_suffix: ${SW_AGENT_IGNORE_SUFFIX:.jpg,.jpeg,.js,.css,.png,.bmp,.gif,.ico,.mp3,.mp4,.html,.svg}
+  # If the operation name of the first span is matching, this segment should be ignored.(multiple split by ",").
+  # Matching rules follow Ant Path match style, like /path/*, /path/**, /path/?.
```

Comment:
Please add more comments about when the user should use `*`, `**`, or `?`.

Reply:
Done.